app/models/cart.rb

```diff
@@ -10,6 +10,10 @@ def add_item(item_id)
     contents[item_id.to_s] += 1
   end
 
+  def update_quantity(item_id, new_quantity)
+    contents[item_id] = new_quantity.to_i
```

Comment:
new_quantity comes in as a string so I changed it to a integer because contents[item_id] is set as a integer. Currently I'm working on how we want to arrange the routes. Right now I redirect_to cart_path (index). I wasn't sure if I should try and render :index instead? The issue here is I send the information through the url which wouldn't go with the user story (which requests current_path to be '/cart' after updating). If we would like to render instead I will look into reconfiguring the routes and adding hidden_fields.